sklearn/mixture/_gaussian_mixture.py

```diff
@@ -492,13 +492,14 @@ class GaussianMixture(BaseMixture):
     n_init : int, default=1
         The number of initializations to perform. The best results are kept.
 
-    init_params : {'kmeans', 'random'}, default='kmeans'
+    init_params : str, default='kmeans'
         The method used to initialize the weights, the means and the
         precisions.
-        Must be one of::
-
-            'kmeans' : responsibilities are initialized using kmeans.
-            'random' : responsibilities are initialized randomly.
+        String must be one of:
+        - 'kmeans' : responsibilities are initialized using kmeans.
+        - 'random' : responsibilities are initialized randomly.
+        - 'random_from_data' : Initial means are randomly selected data points.
+        - 'k-means++' : Use the k-means++ method to initialize.
```

Comment:
Please add a versionchanged:: 1.1 field saying that this params now accepts 'random_from_data' and 'k-means++'.